message_sender/views.py

```diff
@@ -230,6 +243,10 @@ def post(self, request, *args, **kwargs):
                 "metric_name": 'vumimessage.obd.unsuccessful.sum',
                 "metric_value": 1.0
             })
+        outbound_type = "voice" if "voice_speech_url" in \
+            message.metadata else "text"
+        ConcurrencyLimiter.decr_message_count(
+            outbound_type, message.last_sent_time)
```

Comment:
Don't we want to decrement if the call didn't go through though? ie we get an event other than  an ack?